chainreg/chainregistry.go

```diff
@@ -402,13 +402,15 @@ func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
 		// Establish the connection to bitcoind and create the clients
 		// required for our relevant subsystems.
 		bitcoindConn, err := chain.NewBitcoindConn(&chain.BitcoindConfig{
-			ChainParams:        cfg.ActiveNetParams.Params,
-			Host:               bitcoindHost,
-			User:               bitcoindMode.RPCUser,
-			Pass:               bitcoindMode.RPCPass,
-			ZMQBlockHost:       bitcoindMode.ZMQPubRawBlock,
-			ZMQTxHost:          bitcoindMode.ZMQPubRawTx,
-			ZMQReadDeadline:    5 * time.Second,
+			ChainParams: cfg.ActiveNetParams.Params,
+			Host:        bitcoindHost,
+			User:        bitcoindMode.RPCUser,
+			Pass:        bitcoindMode.RPCPass,
+			ZMQConfig: &chain.ZMQConfig{
+				ZMQBlockHost:    bitcoindMode.ZMQPubRawBlock,
+				ZMQTxHost:       bitcoindMode.ZMQPubRawTx,
+				ZMQReadDeadline: 5 * time.Second,
```

Comment:
If we're touching this code, should we extract this into a constant or even make the deadline configurable?

Reply:
sure thing!